packages/test/local-server-stress-tests/src/localServerStressHarness.ts

```diff
@@ -0,0 +1,1268 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
+import { mkdirSync, readFileSync } from "node:fs";
+import path from "node:path";
+
+import { TypedEventEmitter } from "@fluid-internal/client-utils";
+import type {
+	AsyncGenerator,
+	AsyncReducer,
+	BaseFuzzTestState,
+	IRandom,
+	SaveDestination,
+	SaveInfo,
+} from "@fluid-private/stochastic-test-utils";
+import {
+	ExitBehavior,
+	StressMode,
+	asyncGeneratorFromArray,
+	chainAsync,
+	createFuzzDescribe,
+	defaultOptions,
+	done,
+	interleaveAsync,
+	makeRandom,
+	performFuzzActionsAsync,
+	saveOpsToFile,
+	takeAsync,
+} from "@fluid-private/stochastic-test-utils";
+import {
+	type ICodeDetailsLoader,
+	type IContainer,
+	type IFluidCodeDetails,
+} from "@fluidframework/container-definitions/internal";
+import {
+	ConnectionState,
+	createDetachedContainer,
+	loadExistingContainer,
+} from "@fluidframework/container-loader/internal";
+import type { FluidObject } from "@fluidframework/core-interfaces";
+import { unreachableCase } from "@fluidframework/core-utils/internal";
+import {
+	createLocalResolverCreateNewRequest,
+	LocalDocumentServiceFactory,
+	LocalResolver,
+} from "@fluidframework/local-driver/internal";
+import {
+	ILocalDeltaConnectionServer,
+	LocalDeltaConnectionServer,
+} from "@fluidframework/server-local-server";
+import { LocalCodeLoader } from "@fluidframework/test-utils/internal";
+
+import { FuzzTestMinimizer } from "./minification.js";
+import type { MinimizationTransform } from "./minification.js";
+import { createRuntimeFactory, StressDataObject } from "./stressDataObject.js";
+
+const isOperationType = <O extends BaseOperation>(
+	type: O["type"],
+	op: BaseOperation,
+): op is O => op.type === type;
+
+export interface Client {
+	container: IContainer;
+	id: string;
+	entryPoint: StressDataObject;
+}
+
+/**
+ * @internal
+ */
+export interface LocalServerStressState extends BaseFuzzTestState {
```

Comment:
i just copy pasted the existing dds fuzz harness code, and make necessary changes to support whole containers. there isn't very much that is interesting in this file. given how the typing works both here and dds fuzz sharing code could be tricky, but possibly we could share some pieces.